File: practice/calculator.py
n = int(input("Number: "))
m = int(input("Number: "))

print("1 = +, 2 = -, 3 = /, 4 = *")

while True:
    o = int(input("choose your operator: "))
    if o < 0 and o > 4:
        break

    if (o == 1):
        print(f"{m} + {n} = {m + n})")
    if (o == 2):
        print(f"{n} - {m} = {n - m}")
    if (o == 3):
        print(f"{n}/{m} = {n/m}")
    else:
        print(f"{n}*{m} = {n * m}")
    break

File: practice/calculator.c
#include <stdio.h>
#include <ctype.h>

int main (void)
{
    int n = 0; int m = 0;
    {
        printf("Input Two Digits, specify with space: ");
        scanf("%i", &m);
        scanf("%i", &n);
        if (!isdigit(m) || !isdigit(n))
        {
            printf("Not a digit\n");
        }
        else
        {
            return 0;
        }

        int o = 0;
        while (o < 1 || o > 5)
        {
            printf("1 For '+', 2 For '-', 3 For '/', 4 For '*'\n");
            printf("Write the correct no. for your opreator: ");
            scanf("%i", &o);
            if (!isdigit(o))
            {
                return 1;
            }
        }

            switch(o)
            {
                case 1:
                printf("%i + %i = %i\n", m, n, n + m);
                break;
                case 2:
                printf("%i - %i = %i\n", m, n, n - m);
                break;
                case 3:
                printf("%i / %i = %f\n", m, n, (float)m/(float)n);
                break;
                case 4:
                printf("%i * %i = %i\n", m, n, n * m);
                break;
            }
    }
}